types/data.d.ts

```diff
@@ -50,6 +50,7 @@ export interface OopsyData {
   role: Role;
   party: PartyTracker;
   inCombat: boolean;
+  isImmune: (x?: string) => boolean;
```

Comment:
```suggestion
  IsImmune: (x?: string) => boolean;
```

I hate that past me made functions have capital letters in this class, but probably this should be consistent with the other functions here.